gen/dcos-config.yaml

```diff
@@ -315,6 +315,10 @@ package:
       MESOS_GC_DELAY={{ gc_delay }}
       MESOS_HOSTNAME_LOOKUP=false
       GLOG_drop_log_memory=false
+  - path: /etc/dcos-log-config.json
```

Comment:
It looks like there may have been a mistake in insertion here (contains env variables that don't look associated with the service)